File: app/src/main/java/es/etologic/mahjongscoring2/app/model/GamePages.java
package es.etologic.mahjongscoring2.app.model;

public enum GamePages{
    TABLE(0),
    LIST(1);

    private int code;

    GamePages(int code) { this.code = code; }

    public int getCode() { return code; }

    public static GamePages getFromCode(int code) {
        switch (code) {
            default: return TABLE;
            case 1: return LIST;
        }
    }
}

File: app/src/main/java/es/etologic/mahjongscoring2/app/model/GamePages.kt
package es.etologic.mahjongscoring2.app.model

enum class GamePages(val code: Int) {
    TABLE(0),
    LIST(1);
    
    companion object {
        
        fun getFromCode(code: Int): GamePages {
            return when (code) {
                1 -> LIST
                else -> TABLE
            }
        }
    }
}
